plugins/cilium-cni/cilium-cni.go

```diff
@@ -544,7 +544,9 @@ func cmdDel(args *skel.CmdArgs) error {
 		)
 
 		if chainAction.ImplementsDelete() {
-			return chainAction.Delete(context.TODO(), ctx)
```

Comment:
@tgraf Why is CNI plugin returning here immediately? We don't want to perform cleanups below (like removing an endpoint) after running chained plugin(s)? Is there something which I'm missing?

I'm wondering whether I should `return` here or not. So far I removed the `return` and I'm just logging an error.

Reply:
If the chaining implements delete then it will release all resources in that delete command.